src/GlideConversion.php

```diff
@@ -11,29 +11,28 @@
 
 final class GlideConversion
 {
-    /** @var string */
-    private $inputImage;
+    private string $imageDriver = 'gd';
 
-    /** @var string */
-    private $imageDriver = 'gd';
+    private ?string $conversionResult = null;
 
-    /** @var string */
-    private $conversionResult;
+    private string $temporaryDirectory;
 
-    /** @var string */
-    private $temporaryDirectory;
+    public function __construct(private string $inputImage)
```

Comment:
```suggestion
    public function __construct(string $inputImage)
```

Reply:
Constructor property promotion has thrown me off here, using normal properties and some inside the constructor it is harder to keep track. Anyway, not a required change.